### DISCUSSION TITLE
graalvm-jdk: Adding caveat about removing quarantine from installed files

### DISCUSSION THREAD

**Discussion by danielcompton** [Homebrew/discussions#5215] [Casks]
### Output of `brew config`

```shell
brew config
HOMEBREW_VERSION: 4.2.12-61-g7c09aa7
ORIGIN: https://github.com/Homebrew/brew
HEAD: 7c09aa742ccf659f3e4f9eb56c5efc9758099fff
Last commit: 5 days ago
Core tap HEAD: eff2e91e989578384216647ad3765a795a578d22
Core tap last commit: 5 days ago
Core tap JSON: 19 Mar 15:44 UTC
Core cask tap HEAD: 6b991ca41f9d657e2e63ec417452e992f66cb470
Core cask tap last commit: 5 days ago
Core cask tap JSON: 19 Mar 15:44 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_AUTO_UPDATE_SECS: 259200
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vim
HOMEBREW_MAKE_JOBS: 10
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.41.0 => /opt/homebrew/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.3.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.2 => /Applications/Xcode-15.2.0.app/Contents/Developer
Rosetta 2: false
```

### Output of `brew doctor`

```shell
.
```

### Description of issue

In https://github.com/graalvm/homebrew-tap#macos-catalina-specifics it says:

> On macOS Catalina, you may get a warning that "the developer cannot be verified". This is due to GraalVM not being signed and notarized yet. The check, however, can be disabled in the "Security & Privacy" preferences pane or by running the following command:
>
> xattr -r -d com.apple.quarantine /Library/Java/JavaVirtualMachines/graalvm-jdk-XX.Y.Z

Would it be possible to add this to the caveats in the graalvm-jdk cask? You can't use the cask until you remove the quarantine, but you get unclear error messages saying the file is damaged.

https://stackoverflow.com/questions/65786439/problem-with-installing-running-graalvm-on-mac
https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/g/graalvm-jdk.rb

### Relevant casks

graalvm-jdk

**Comment by SMillerDev** [comment]
First of, that is a tap controlled by GraalVM so they would have to add that. But if it wasn't the answer would still be "no" since Homebrew does not encourage disabling macOS security features

**Reply by danielcompton** [reply]
Sorry if I wasn't clearer, I was referring to https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/g/graalvm-jdk.rb, the built-in Homebrew Cask.
